mlrun/utils/helpers.py

```diff
@@ -406,13 +407,18 @@ def _set_artifact_path(task):
     return conf
 
 
-def tag_image(base: str):
-    ver = config.images_tag or config.version
-    if ver and (
-        base == "mlrun/mlrun" or (base.startswith("mlrun/ml-") and ":" not in base)
-    ):
-        base += ":" + ver
-    return base
+def enrich_image(image: str):
+    tag = config.images_tag or mlrun.utils.version.get_version_info()["version"]
+    registry = (
+        config.images_registry
+        or mlrun.utils.version.get_version_info()["docker_registry"]
```

Comment:
docker_registry should be taken from the config directly, not from version (it's not related to the version this was built with)